config/config-mp/src/main/java/io/helidon/config/mp/MpSystemPropertiesMetaConfigProvider.java

```diff
@@ -42,4 +43,9 @@ public List<? extends ConfigSource> create(String type, Config metaConfig, Strin
     public int priority() {
         return 300;
     }
+
+    @Override
+    public ConfigSource create(Reader content) {
+        return MpConfigSources.systemProperties();
```

Comment:
And this is the case where it should just not implement this method, as it does not make any sense. System properties config source is not suitable for content, and should throw an exception (this is a user error)